config.example.json

```diff
@@ -2,7 +2,7 @@
     "rabbit_url": "http://127.0.0.1:15672",
     "rabbit_user": "guest",
     "rabbit_pass": "guest",
-    "publish_port": "9419",
+    "publish_port": "9149",
```

Comment:
Can you please revert this change.

Reply:
sorry, I didn't get the message in time。I'll commit PR again

Reply:
https://github.com/kbudde/rabbitmq_exporter/pull/254